pipert/utils/visualizer/video_visualizer.py

```diff
@@ -110,7 +110,7 @@ def draw_instance_predictions(self, frame, predictions):
         """
 
         frame_visualizer = Visualizer(frame, self.metadata)
-        if not predictions.has("pred_boxes"):
+        if not predictions.has("pred_boxes") or not predictions.pred_boxes:
```

Comment:
Shouldn't `not predictions.has("pred_boxes")` and `not predictions.pred_boxes` have the same output, or am I missing something? @sionovd 
I think that if `predictions` doesn't have `pred_boxes` as attribute the latter will raise an Exception.

Reply:
I often encountered an error in this area, where `not predictions.has("pred_boxes")` returned true, but `predictions.pred_boxes` was empty, which is why I added the second condition. 
And regarding your second point - doesn't python do a lazy evaluation of OR, meaning that it wouldn't reach the second part if there is no `pred_boxes` attribute? @ItamarWilf

Reply:
Good point.
We can leave that fix for now but we should check what is the cause of this error..